src/NLog/Targets/PerformanceCounterTarget.cs

```diff
@@ -234,7 +243,8 @@ protected override void Write(LogEventInfo logEvent)
         {
             if (this.EnsureInitialized())
             {
-                this.perfCounter.Increment();
+                long incrementValue = Int64.Parse(IncrementValue.Render(logEvent));
```

Comment:
Please use tryparse and write to the internallog in case of a parse failure. I think Error is the right level?